client/src/components/Nav/NavProfile.jsx

```diff
@@ -11,12 +11,20 @@ class NavProfile extends Component {
     else return '/person.jpg'
   }
 
+  constuctor() {
+    this.routeChange = this.routeChange.bind(this);
+   }
+
+   routeChange(){
```

Comment:
I know better way to do this, Sorry but seems really bad

Reply:
Care to elaborate?

Reply:
`<li><Link to={"/profile/" + sessionStorage['User-id']}>` seems better and also takes less code and resources